php/composer.json

```diff
@@ -6,10 +6,10 @@
   "homepage": "https://developers.google.com/protocol-buffers/",
   "license": "BSD-3-Clause",
   "require": {
-    "php": ">=5.5.0"
+    "php": ">=7.0.0"
```

Comment:
This drops support for 5.5 and 5.6 in the native PHP library. Was this intentional? I see no mention of this anywhere in the PR or README.